main.py

```diff
@@ -136,8 +136,9 @@ def _init_logging():
     config = dict(
         name = "Shallow scrape with threading",
         description = "Scrapes the entire catalog using multiple threads",
-        threads = 4,
+        threads = 1,
```

Comment:
still does, I just thought that 4 was a weird preset. 1 is better for understanding what's going on, 10-20 is better for actually running a full scrape I think.